docs/whats-new.mdx

```diff
@@ -19,6 +20,8 @@ Target Release Date: April, 2024
 
 **@loaders.gl/parquet**
 
+- Support compression modes: 'zstd', 'snappy', 'gzip', 'brotli', 'no_compression'.
+- Geoparquet: support large 'Polygons' and 'MultiPolygons'. Up to 68159 vertices for 'MultiPolygon' and 20062 vertices for 'Polygon' has been tested.
```

Comment:
It is a little odd to specify such numbers, it sounds like we don't understand what the limits in our implementation are.

Reply:
"Geoparquet: support large 'Polygons' and 'MultiPolygons' (> 10,000 vertices)"

Is this better?

Reply:
Actually, threshold was 65536 vertices (16 bit)

Reply:
What confused me is that we reverted the Uint16 fix.

Reply:
You reverted but then you've fixed it back in https://github.com/visgl/loaders.gl/pull/2940 . That was lucky change in another place )

Reply:
![image](https://github.com/visgl/loaders.gl/assets/18549629/7356c98f-5b31-4b82-8c97-aad44d94c5c0)

Reply:
So the reverted PR is not neede anymore